mundane-bytes/src/main/scala/com/ambiata/mundane/bytes/Buffer.scala

```diff
@@ -0,0 +1,26 @@
+package com.ambiata.mundane.bytes
+
+case class Buffer(bytes: Array[Byte], offset: Int, var length: Int) {
```

Comment:
Would consider adding skull and cross bones. Needs to be clear this isn't a ByteVector type of thing, i.e. we are trying to bring convenience to crazy town, not correctness.